packages/modal/addon/components/nucleus-modal/dialog.js

```diff
@@ -167,12 +167,13 @@ class Dialog extends Component.extend(scroll) {
   didInsertElement() {
     super.didInsertElement(...arguments);
     this.getOrSetTitleId();
-    this.bindScrolling('.nucleus-modal__body');
+    let _scrollCallback = bindEvent(this, 'scroll', this.scrolled, '.nucleus-modal__body');
```

Comment:
Personal opinion, to use named exports and use as:

```js
let _scrollCallback = Util.bindEvent(this, 'scroll', this.scrolled, '.nucleus-modal__body');
```
Just to make it blatantly obvious that it's a util and not a function in the class.